.github/workflows/build.yml

```diff
@@ -55,6 +55,14 @@ jobs:
           docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
           docker-token: ${{ secrets.DOCKERHUB_TOKEN }}
       #NB: We restore/save cache manually so that we save the cache even if the build fails
+      - name: Load docker build cache
```

Comment:
I: Just pushed this change up in light of the recent build starting from scratch. It looks like since the Docker cache is saved last, GH purges caches from oldest to newest, and that the docker cache is larger than the limit it's causing all of our caches to be cleared. (It's also possible someone manually deleted, but making this change just to be safe.)

We likely just need to switch to the s3 caching action so we have no size limit.